Libraries/Image/resolveAssetSource.js

```diff
@@ -58,22 +59,27 @@ function getOfflinePath() {
   return _offlinePath;
 }
 
+function getResourceIdentifier(asset) {
+  var assetDir = getBasePath(asset);
+  // E.g. 'assets_awesomemodule_icon'
+  // The Android resource system picks the correct scale.
+  return (assetDir + '/' + asset.name)
+    .toLowerCase()
```

Comment:
This logic, and the logic in `getAssetDestPathAndroid` is shared.
I'm not sure if we'd rather have this depend on `local-cli`, or the other way round.
cc @frantic @martinbigio 

We definitely shouldn't use two different copies of the same code in one file though. `getPathInArchive` uses this copy, while `getScaledAssetPath` uses the other.